### PR TITLE
Update WCT to 7.0

### LINKED ISSUE #3986: Update Windows Community Toolkit dependencies to 7.0

### PR DESCRIPTION
**Pull request by d2dyno1**
Closes #3986

### CODE REVIEW

**Comment by d2dyno1** [comment]
**Help Wanted!!**
image

File: Files/Files.csproj [hunk 1]
```diff
@@ -986,28 +986,28 @@
       <Version>7.0.0-preview4</Version>
```
Comment (michael-hawker) [inline Files/Files.csproj:986]: You need to update all the packages together.

File: Files/Files.csproj [hunk 2]
```diff
@@ -986,28 +986,28 @@
...
     </PackageReference>
     <PackageReference Include="Microsoft.Toolkit.Uwp.DeveloperTools">
-      <Version>6.1.1</Version>
+      <Version>7.0.0</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.Toolkit.Uwp.Notifications">
-      <Version>6.1.1</Version>
+      <Version>7.0.0</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.Toolkit.Uwp.UI">
```
Comment (michael-hawker) [inline Files/Files.csproj:997]: You used to get the Behaviors included with this package, now you don't so you need to add the Behaviors package explicitly to your dependencies. https://www.nuget.org/packages/Microsoft.Xaml.Behaviors.Uwp.Managed/

**Review by michael-hawker — CHANGES_REQUESTED** [review]

File: Files/Files.csproj (hunk 2 above)
Comment (yaichenbaum) [reply Files/Files.csproj:997]: Thank you! I didn't realize that there used to be another dependency.

File: Files/Files.csproj (hunk 1 above)
Comment (yaichenbaum) [reply Files/Files.csproj]: Resolved with bc21392c73043e9e98d9608bf9aaa7a8d0d83c55.

File: Files/UserControls/Widgets/DrivesWidget.xaml [hunk 3]
```diff
@@ -5,7 +5,7 @@
     xmlns:animations="using:Microsoft.Toolkit.Uwp.UI.Animations"
     xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-    xmlns:extensions="using:Microsoft.Toolkit.Uwp.UI.Extensions"
+    xmlns:extensions="using:Microsoft.Toolkit.Uwp"
```
Comment (tsvietOK) [inline Files/UserControls/Widgets/DrivesWidget.xaml:8]:
```suggestion
    xmlns:extensions="using:Microsoft.Toolkit.Uwp.UI"
```

File: Files/UserControls/Widgets/LibraryCards.xaml [hunk 4]
```diff
@@ -5,7 +5,7 @@
     xmlns:animations="using:Microsoft.Toolkit.Uwp.UI.Animations"
     xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-    xmlns:extensions="using:Microsoft.Toolkit.Uwp.UI.Extensions"
+    xmlns:extensions="using:Microsoft.Toolkit.Uwp"
```
Comment (tsvietOK) [inline Files/UserControls/Widgets/LibraryCards.xaml:8]:
```suggestion
    xmlns:extensions="using:Microsoft.Toolkit.Uwp.UI"
```

**Comment by tsvietOK** [comment]
Change ReorderGridAnimation.Duration to ItemsReorderAnimation.Duration

**Comment by tsvietOK** [comment]
NavigationToolbar.xaml: Mouse.Cursor to FrameworkElementExtensions.Cursor

**Comment by d2dyno1** [comment]
> NavigationToolbar.xaml: Mouse.Cursor to FrameworkElementExtensions.Cursor

Hey @tsvietOK. Could you commit that? I can't do it right now

**Comment by d2dyno1** [comment]
And thanks for the help :+1:

**Comment by tsvietOK** [comment]
image
Now it builds successful, but it crashes when trying to access to drive

**Comment by d2dyno1** [comment]
@tsvietOK maybe it requires different data type?

**Comment by tsvietOK** [comment]
Fixed

File: Files/BaseLayout.cs [hunk 5]
```diff
@@ -899,7 +909,7 @@ await ParentShellPageInstance.FilesystemViewModel.GetFolderFromPathAsync(item.It
         }
 
         private ListedItem dragOverItem = null;
-        private DispatcherTimer dragOverTimer = new DispatcherTimer();
+
```
Comment (yaichenbaum) [inline Files/BaseLayout.cs:912]:
```suggestion
```

**Review by yaichenbaum — APPROVED** [review]
Great work! LGTM

**Pull request merged**